roles/selinux_verify/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+# vim: set ft=ansible:
+#
+- name: Verify that the SELinux context is correct on ostree/rpm-ostree
+  shell: ls -Z {{ item }} | awk '{print $4}'
+  register: selinux_context
+  failed_when: "'install_exec_t' not in selinux_context.stdout"
+  with_items:
+    - /usr/bin/ostree
+    - /usr/bin/rpm-ostree
+    - /usr/libexec/rpm-ostreed
+  when: ansible_distribution != "Fedora"
+
+
+- name: Verify that the SELinux context is correct ostree/rpm-ostree (Fedora)
+  shell: ls -Z {{ item }} | awk '{print $1}'
+  register: selinux_context_f
+  failed_when: "'install_exec_t' not in selinux_context_f.stdout"
+  with_items:
+    - /usr/bin/ostree
+    - /usr/bin/rpm-ostree
+    - /usr/libexec/rpm-ostreed
+  when: ansible_distribution == "Fedora"
```

Comment:
How about `getfattr -n security.selinux tmp.txt --only-values` instead? That should be the same on all platforms and doesn't need any parsing.